### ISSUE TITLE
Is there anyone who didn't get banned?

### ISSUE THREAD

**Issue by mvp256** [NoxxDev/NecroBot#446]
Now we know a lot of people were banned, so how people get banned is not important.
I wanna check it if there were any survivor...

**Comment by vkeydevlz** [comment]
yes level 21

**Comment by popsoda27** [comment]
still surviving fine on lvl 27

**Comment by mvp256** [comment]
How do you guys do that? Thanks.

**Comment by Paraiso0319** [comment]
popsoda27, when did you create your account? mine was Aug 15, and it was banned 14 Hours ago.

**Comment by popsoda27** [comment]
@Paraiso0319 oh im not too sure exactly when..unless there is a way to check cos its a google acct. If im not wrong it's sometime on 10th Aug

**Comment by Paraiso0319** [comment]
@popsoda27 , You can check it at your PoGo account, when you click the Face/Level of your character, it will show you the creation date of your account. anyway, you're one lucky player. haha

**Comment by kalamera** [comment]
did any google account got banned at all?

**Comment by popsoda27** [comment]
@Paraiso0319 oh i was mistaken, i started mine on 19th. Maybe its just a matter of time

**Comment by imolesi** [comment]
My fourth account just got banned, was level 22: never sniped, sometimes in the past got soft ban for speed, used bot and mobile to play with...

**Comment by vkeydevlz** [comment]
i also started on 19th august i used ptc account

**Comment by memiedeedy** [comment]
Just got my 5th one banned this morning
So decide to stop Pokemon for now

**Comment by bulubebek** [comment]
six accounts, all banned. niantic wins for now.

**Comment by ramkissoon11** [comment]
3rd account gone

**Comment by mvp256** [comment]
Has anyone ever tried "GO Simulator"?

**Comment by g0ku123** [comment]
4 account. All banned. Any other solutions to level up?

**Comment by HenryWait** [comment]
I am not banned (yet), I used GPS spoofing on android early on and didn't get banned for that, ingress account did though and I didn't even use it in that. 
I used necro bot latest version day of release and yesterday for approx 3 hours 3 times, no ban yet.

**Comment by suhail09s** [comment]
I had two accounts(31,24) and were banned with the first wave,which I spoofed,snipped,botted with realistic leaps and speeds.(with the original necrobot and other bots)

now I have 2 accounts(14,21),still running 20 mins ago,one of them botted( in Time Square) and snipped crazy but with  average speeds,the other botted with slow speeds and around one place in my city.(with this bot only)

I hope that was helpful.thx for the hard work

**Comment by impisi1229** [comment]
@mvp256 My 8/14 bot acc got banned. Starting using GO Simulator with nox, waiting for the result until next ban wave.

**Comment by whiskey2wine** [comment]
My account level 21 now. Bot all day long with other program to close and start the bot at random time with safe setting. Not ban yet!

(I also sniped Pokemon)

**Comment by REDESTOR** [comment]
post your settings for us...i think they have to check what information the server is retrieving when we play and what info we are sending with the bot...or maybe niantic just spotted the behavior pattern of the bot and flag us

**Comment by REDESTOR** [comment]
for example the recycling it always leave the same amount of things, i got a level 30 account by playing, using the bot, and using nox 60%real playing And 40%"playing" without any safe config, when i stop playing and start just "playing" i got banned

**Comment by ausscheider** [comment]
i go back to old ;)
i test now v.0.8.6 without any humanlike
200k EP/h
150 km/h
if i banned with human like or without? what happens :D

**Comment by fence1247** [comment]
I had 6-7 accounts. About 2/3 were sniping accounts. All of my accounts but one account was banned, including one 26, one 28, and one 32.

I have a level 22 account which did plenty of sniping (dragonites, snorlax, lapras galore) which I was surprised to find was still not banned. But my no sniping accounts were all banned :(

**Comment by mvp256** [comment]
@fence1247 How did you snipe for them?

**Comment by Skolzack** [comment]
No matter what you use or configs, we don't know how Niantic is banning us..

**Comment by hardknoxni69a** [comment]
Its probably the simple fact were all on rocketapi which is open source until a new api comes out there probably gonna get us everytime regardless of how himanlile

**Comment by fence1247** [comment]
@mvp256 using poke feeder, just with very defined settings for only high power pokemon as targets, with a 60000 delay between snipes. But I only sniped on the account prior to August 10th.

**Comment by Skolzack** [comment]
That's the problem of open source: They can download it, analyze it and drop the banhammer lol

**Comment by mvp256** [comment]
@fence1247 I think the rocket API issue is still exist in any modified version of Necrobot? Maybe we need some time to prove if it works or not.

**Comment by fence1247** [comment]
Yes RocketAPI is in the modified version. The ultimate working bot would be a bot worked into the code of "PokeGo ++" (a jailbreaked Pokemon GPS spoofer) because it doesn't modify the code of pokemon go or sign the app at all when installed on the iPhone. It modifies bits and plies of the code as it goes along.

-as a side note, the program Insta-Pokego! does not appear to use RocketAPI and there users have had very low ban rates.

User Blastophere summed it up pretty good for reasons why a bot might not work:
#1 Bot Does not create a Supported mobile device that can run Pokemon go. So niantic detects using your using unsupported device = Ban incoming.
#2 Bot Does not hack into mobile providers network to show niantic you are using a providers Mobile Network in the area you are botting so that = ban incoming.
#3 Their no way the bot can spoof a location in the area detect someone free WiFi and connect to it. If it did it will epic. So ban incoming
#4 Remember you have to give permission to niantic to use your camera and photos. Idk if the bot automatically uses Ar mode or not. If it does not that = Ban incoming cause it shows niantic your catching Pokemon in a black screen.
#5 Also since u can't connect or hack into a providers mobile network. Niantic can just search were I.P address is coming from and sees your spoofing = ban incoming.

If all these issues can be addressed you will have a the best bot ever.

**Comment by JoNMii** [comment]
Good Words, my feelings @fence1247

**Comment by langit7** [comment]
@fence1247 
2. POGO clien did not ask access permit to that info. to check imei and phonenumer Android app not to ask permit ,
3-5. IP is not belong to 1 location, i can setup US IP in like Singapore , just setup the AS there or create some tunnel, its normal some big company to use head office IP block

**Comment by Paraiso0319** [comment]
@impisi1229 Hey, i'm also testing Go Simulator with Nox. i started 5 hours ago. any news from your dummy account? Thanks.

**Comment by MV1977** [comment]
I'm also sure that Niantic is not spending half their resources on trying to identify bot's. So I'm assuming (keep in mind, assumption is the mother of all fuck-ups) they are not looking at your IP address, whether or not you are using a mobile phone provider (think MiFi devices) or to which WiFi you are connected. 

Without any knowledge of the communication between PokemonGO and Niantic, I do expect there is something in the communication that allows Niantic to filter out the bot accounts. Maybe it has something to do with the UK6 which makes it detectable, maybe it has something to do with the amount of inventory updates it does, or maybe it has something to do with the API throttling messages I see every now and then. (as said, I have no programming experience, nor any knowledge of the comms between PokemonGO and Niantic)

So far, every account I have used for botting has been banned at some point in time (some even a week after the last time I used the bot). Independent of the config I used, from extremely conservative (bot used for max 1 hour a day with very low speeds, no evolves, small area, in my hometown, etc., for 5 days, rest manual play) to extreme sniping with high speed walking, hopping across the globe throughout the day, etc.

I'm sure there's a Niantic developer (or ex-developer) somewhere on Reddit who knows the answer :).
